libcodechecker/server/config_database.py

```diff
@@ -4,41 +4,37 @@
 #   License. See LICENSE.TXT for details.
 # -------------------------------------------------------------------------
 """
-Database server handling.
+Database server handling for the server's configuration database.
```

Comment:
My main concern, this way we always need to remember to change the code in multiple places. The information you mentioned is not lost, you can always check how the database engine was created (or add debug prints).